packages/shared/src/hooks/usePostMenuActions.ts

```diff
@@ -2,14 +2,23 @@ import { useMutation } from 'react-query';
 import { useMemo } from 'react';
 import { PromptOptions, usePrompt } from './usePrompt';
 import { deletePost, Post } from '../graphql/posts';
+import { SourcePermissions, SourceType } from '../graphql/sources';
+import { Roles } from '../lib/user';
+import { useAuthContext } from '../contexts/AuthContext';
 
 interface UsePostMenuActions {
   onConfirmDeletePost: () => Promise<void>;
 }
 
+interface DeletePostProps {
+  id: string;
+  index?: number;
+}
+
 interface UsePostMenuActionsProps {
   post: Post;
-  onPostDeleted?: () => void;
+  postIndex?: number;
+  onPostDeleted?: (args: DeletePostProps) => void;
 }
 
 const deletePromptOptions: PromptOptions = {
```

Comment:
title should be: `Delete post?`
description: `Are you sure you want to delete this post?`
button color should be cabbage.